index.html

```diff
@@ -585,7 +585,13 @@ <h3>Validating an ExposedThingInit</h3>
             <li>if |key| |value| is a <code>string</code> then if |value| is equal to one of the elements in |optional| remove |key| from |schema|</li>
           </ol>
         </li>
-        <li>Return the result of applying |schema| to |init|</li>
+        <li>Return the result of <a>validating</a> |init| with |schema|
+          <p class="ednote"><dfn>Validating</dfn> steps are still under discussion.
+            Currently this specification reference to the validation process of JSONSchema. Please
+            follow this <a href="https://json-schema.org/draft/2019-09/json-schema-validation.html">document</a> 
+            when validating |init| with |schema|. Notice that the working group is evaluating an alternative formal approach.
+          </p>
```

Comment:
Then also this needs some update:

```
<p class="ednote"> 
  The<dfn>validating an object with JSON Schema</dfn> steps are still under discussion.
  Currently this specification uses the <a href="https://json-schema.org/draft/2019-09/json-schema-validation.html">validation process of JSON Schema</a>. The working group is evaluating an alternative formal approach.
</p>
```

Alternatively, we could already factor out the algo in a subsection, just having an empty body with an ednote, like this:

```
To <dfn>validate an object with JSON Schema</dfn>, given |object| and |schema|, run the following steps:
<ol class="algorithm"
  <li>
     Return the result of <a href="https://json-schema.org/draft/2019-09/json-schema-validation.html">validating</a>
     |init| using |schema|.
     <p class = "ednote">
         ...
     </p>
  </li>
</ol>
```

Reply:
Thank you for the input! I kept the editor's note because I was not sure where to create the section. We could still add it later.